test/k8sT/Services.go

```diff
@@ -146,4 +147,162 @@ var _ = Describe("K8sServicesTest", func() {
 			net.JoinHostPort(data.Spec.ClusterIP, fmt.Sprintf("%d", data.Spec.Ports[0].Port)))
 		testHTTPRequest(url)
 	})
+
+	It("CNP Specs Test", func() {
+
+		// Various constants used in this test
+		wgetCommand := "%s exec -t %s wget -- --tries=2 --connect-timeout 10 %s"
+
+		version := "version"
+		v1 := "v1"
+		v2 := "v2"
+
+		productPage := "productpage"
+		reviews := "reviews"
+		ratings := "ratings"
+		details := "details"
+		app := "app"
+		bookinfo := "bookinfo"
+		health := "health"
+
+		apiPort := "9080"
+
+		podNameFilter := "{.items[*].metadata.name}"
+
+		// shouldConnect asserts that srcPod can connect to dst.
+		shouldConnect := func(srcPod, dst string) {
+			By(fmt.Sprintf("Checking that %s can connect to %s", srcPod, dst))
+			res := kubectl.Exec(fmt.Sprintf(wgetCommand, helpers.KubectlCmd, srcPod, dst))
+			res.ExpectSuccess()
```

Comment:
Could you add a message in this ExpectSuccess? If fails we need to know from where to where it fails.

Reply:
Fixed.